src/frequenz/client/common/metric/__init__.py

```diff
@@ -57,7 +57,16 @@ class Metric(Enum):
     AC_POWER_FACTOR_PHASE_2 = PBMetric.METRIC_AC_POWER_FACTOR_PHASE_2
     AC_POWER_FACTOR_PHASE_3 = PBMetric.METRIC_AC_POWER_FACTOR_PHASE_3
 
-    # AC energy metrics
+    """AC energy metrics
+    * This energy metric is reported directly from the component, and not a
+    result of aggregations in our systems. If a component does not have this
+    metric, this field cannot be populated.
+    * Components that provide energy metrics reset this metric from time to
+    time. This behaviour is specific to each component model. E.g., some
+    components reset it on UTC 00:00:00.
+    * This energy metric does not specify the timestamp since when the energy
+    was being accumulated, and therefore can be inconsistent.
+    """
```

Comment:
As it is, this docstring is documenting the varaible `AC_POWER_FACTOR_PHASE_3` (docstrings always document the symbol above, not below it).

Since we have multiple energy metrics, I think it should be best to document this in the `Enum` class, and in the metric just say you have to be careful and to read the `Enum` docs, so we don't have a big explanation duplicated in many metrics.

You can always check how generated docs look like by doing `pip install -e .[dev]; mkdocs serve`